README.md

```diff
@@ -1584,6 +1587,63 @@ Server: spray-can/1.3.3
 }
 ```
 
+## POST /api/workflows/:version/query
+
+This endpoint allows for querying workflows based on the same criteria
+as [GET /api/workflows/:version/query](#get-apiworkflowsversionquery).
+
+Instead of specifying query parameters in the URL, the parameters
+must be sent via the POST body. The request content type must be
+`application/json`. The json should be a list of objects. Each json
+object should contain a different criteria.
+
+cURL:
+
+```
+$ curl -X POST --header "Content-Type: application/json" -d "[{\"start\": \"2015-11-01\"}, {\"end\": \"2015-11-03\"}, {\"status\": \"Failed\"}, {\"status\": \"Submitted\"}]" "http://localhost:8000/api/workflows/v1/query"
+```
+
+HTTPie:
+
+```
+$ echo "[{\"start\": \"2015-11-01\"}, {\"end\": \"2015-11-03\"}, {\"status\": \"Failed\"}, {\"status\": \"Submitted\"}]" | http "http://localhost:8000/api/workflows/v1/query"
+```
+
+Response:
+```
+HTTP/1.1 200 OK
+Content-Length: 133
+Content-Type: application/json; charset=UTF-8
+Date: Tue, 02 Jun 2015 18:06:56 GMT
+Server: spray-can/1.3.3
+
+{
+  "results": [
+    {
+      "name": "w",
+      "id": "fdfa8482-e870-4528-b639-73514b0469b2",
+      "status": "Succeeded",
+      "end": "2015-11-01T07:45:52.000-05:00",
+      "start": "2015-11-01T07:38:57.000-05:00"
+    },
+    {
+      "name": "hello",
+      "id": "e69895b1-42ed-40e1-b42d-888532c49a0f",
+      "status": "Succeeded",
```

Comment:
The request above filter by `Failed` and `Submitted` so this probably shouldn't be here